### PR TITLE
[Release] SQL archiving and release patch + composer update

### PR DESCRIPTION
**Pull request by ridz1208**
### Brief summary of changes
Preparation for release 21

concatenation bash command:
```
for n in SQL/New_patches/*.sql; do echo $n; echo -e "\nSELECT 'Running: $n';\n" >> SQL/Release_patches/20.3_To_21.0_upgrade.sql; cat $n >> SQL/Release_patches/20.3_To_21.0_upgrade.sql; done;

```

### CODE REVIEW

File: SQL/Release_patches/20.3_To_21.0_upgrade.sql [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+    'boolean',
+    ID,
+    'Enable "Pwned Password" check',
+    22
+  FROM
+    ConfigSettings
+  WHERE
+    Name="study";
+
+INSERT INTO Config (ConfigID, Value) SELECT ID, 'true' FROM ConfigSettings WHERE Name="usePwnedPasswordsAPI";
```
Comment (xlecours) [inline SQL/Release_patches/20.3_To_21.0_upgrade.sql]: This should be set to false. I don't think that all instances should send HTTP request to a server in Netherlands at each password changes.

**Comment by xlecours** [comment]
Travis is mad; `modules/issue_tracker/test/Issue_TrackerTest.php::testClearFormIssueTracker` is all commented

File: SQL/Release_patches/20.3_To_21.0_upgrade.sql (hunk 1 above)
Comment (ridz1208) [reply SQL/Release_patches/20.3_To_21.0_upgrade.sql]: hmm... I guess it was approved by someone.

I track that and see if it was previously discussed

Comment (driusan) [reply SQL/Release_patches/20.3_To_21.0_upgrade.sql]: It was discussed.. @xlecours is right, it's probably just that the schema was reviewed/changed and the patch was missed.

Comment (driusan) [reply SQL/Release_patches/20.3_To_21.0_upgrade.sql]: I was mistaken, the discussion was about whether to include a config setting, not what the default should be.

**Pull request merged**